reclist/metrics/distance_metrics.py

```diff
@@ -21,9 +25,12 @@ def error_by_cosine_distance(model, y_test, y_preds, k=3, bins=25, debug=False):
         if m['Y_PRED']:
             vector_test = model.get_vector(m['Y_TEST'][0])
             vector_pred = model.get_vector(m['Y_PRED'][0])
-            if vector_pred and vector_test:
-                cos_dist = cosine(vector_pred, vector_test)
-                cos_distances.append(cos_dist)
+            try:
+                if vector_pred is not None and vector_test is not None:
+                    cos_dist = cosine(vector_pred, vector_test)
+                    cos_distances.append(cos_dist)
+            except ValueError as e:
+                print(e)
```

Comment:
The `error_by_cosine_distance` metric was already implemented, right? Should we add a new custom metric instead of modifying theirs, do you think this is a good point at all? I was just thinking that in case we merge and PR to their repo, it'd be good to just add things instead of having modified theirs.

Reply:
That's a good point, I had changed it for debugging purposes, but forgot to restore the original code. Done.